xbmc/games/addons/GameClientMouse.cpp

```diff
@@ -114,7 +112,7 @@ void CGameClientMouse::OnButtonRelease(const std::string& button)
 
   event.type                   = GAME_INPUT_EVENT_DIGITAL_BUTTON;
   event.port                   = GAME_INPUT_PORT_MOUSE;
-  event.controller_id          = CONTROLLER_NAME;
+  event.controller_id          = MOUSE_CONTROLLER_ID;
   event.feature_name           = button.c_str();
```

Comment:
Create a static MouseControllerID in CMouseWindowingButtonMap, or have a default constructed instance of CMouseWindowingButtonMap in CGameClientMouse?